### PR TITLE
Sleep sort ;)

### PR DESCRIPTION
**Pull request by Sumith1896**

### CODE REVIEW

**Comment by Sumith1896** [comment]
@aktech Any issues? 
This is ready to merge

File: pydsa/sleep_sort.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    """
+    sleep_sort.result = []
+    def add1(x):
+        sleep_sort.result.append(x)
+    mx = a[0]
+    for v in a:
+        if mx < v: mx = v
+        Timer(v, add1, [v]).start()
+    sleep(mx+1)
```
Comment (aktech) [inline pydsa/sleep_sort.py]: PEP8: space around operator

**Comment by Sumith1896** [comment]
@aktech Sorry for the delayed response. It is now fixed.

**Comment by aktech** [comment]
@Sumith1896  Thanks for fixing. Merging.

**Pull request merged**